Packs/GoogleApigee/Integrations/GoogleApigeeEventCollector/GoogleApigeeEventCollector_test.py

```diff
@@ -0,0 +1,189 @@
+import pytest
+
+from GoogleApigeeEventCollector import (
+    Client,
+    fetch_events,
+    get_events_command,
+)
+
+
+def mock_client():
+    return Client(base_url='https://test.com', verify=False, proxy=False, org_name='org', username='user',
```

Comment:
Since in both places where you use this mocker you add `mocker.patch.object(Client, 'get_access_token', return_value={'access_token': 'access_token'})`, save the client to a variable and do the mocking here before you return it.